### ISSUE TITLE
indicator didn't show!

### ISSUE THREAD

**Issue by devabir93** [IslamKhSh/CardSlider#12]
Hello, thank you for your nice lib, but I am having a problem that is indicator didn't show.
here is my snippet code.
```
            <com.github.islamkhsh.CardSliderViewPager
                android:id="@+id/viewPager"
                android:layout_width="match_parent"
                android:layout_height="@dimen/_150sdp"
                android:layout_marginTop="10dp"
                app:auto_slide_time="5"
                app:cardSlider_indicator="@+id/indicator"
                app:cardSlider_otherPagesWidth="24dp"
                app:cardSlider_pageMargin="12dp"
                app:cardSlider_smallScaleFactor="1" />

            <com.github.islamkhsh.CardSliderIndicator
                android:id="@+id/indicator"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:layout_gravity="center"
                android:layout_marginTop="@dimen/_8sdp"
                app:defaultIndicator="@drawable/ic_slider_unselected"
                app:indicatorsToShow="3"
                app:selectedIndicator="@drawable/ic_slider_selected" />

```
i think there is nothing to do in java code, right?

**Comment by IslamKhSh** [comment]
There is no errors in your snippet so I think that the problem is in your parent ViewGroup ... make sure to set constrains to indicator if you use ConstraintLayout or RelativeLayout and also make sure that's the orientation is vertical if you use LinearLayout.

**Comment by devabir93** [comment]
I am sure that everything is ok, actually, when I put a dummy data it showed but when I changed it to show from API it disappears!

**Comment by IslamKhSh** [comment]
fixed in v1.0.1
